public/views/partials/footer.html

```diff
@@ -0,0 +1,17 @@
+<footer id="footer">
+	<div class="scroll-to-top">
+		<a href="" onclick="scrollToTop();">Top</a>
+	</div>
+	<div class="footer-container">
+		<ul class="footer-link-collection">
+            <li><a class="footer-link" href="/contact">Contact Us</a></li>
+            <li><a class="footer-link" href="/media">Social Media</a></li>
+            <li><a class="footer-link" href="/sponsors">Sponsors</a></li>
+            <li><a class="footer-link" href="/links">Links</a></li>
+        </ul>
+        <hr id="footer-bar">
+        <div id="legal-info">
+        	<p class="text">&copy 2016 Harvard Undergraduate Robotics Club. All Rights Reserved.</p>
```

Comment:
You don't have a class called text in css. Use my-text instead for every div that contains text in order to use our fonts.